### PR TITLE
[ADD] delivery_dropoff_site module

### PR DESCRIPTION
**Pull request by bealdav**
Manage features related to drop-off sites

Main international carriers provide transportation services to specific areas managed by them or by subcontractors.

Then, recipients comes pick up their packages in these sites

### CODE REVIEW

File: delivery_dropoff_site/__init__.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+#  Copyright (C) 2014 Akretion (http://www.akretion.com).
+#  @author David BEAL <david.beal@akretion.com>
+#
+##############################################################################
+
+
+import stock  # noqa
+import file_exchange    # noqa
+import partner          # noqa
+
```
Comment (gurneyalex) [inline delivery_dropoff_site/__init__.py]: use `from . import xxxx`, remove noqa comments

File: delivery_dropoff_site/__openerp__.py [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+    'description': """
+Delivery Drop-off Site
+======================
+
+Manage features related to drop-off sites
+-----------------------------------------
+
+Main international carriers provide transportation services to specific areas managed by them or by subcontractors.
+
+Then, recipients comes pick up their packages in these sites
```
Comment (gurneyalex) [inline delivery_dropoff_site/__openerp__.py]: s/comes/come and /

you may want to add a '.' at the end of the sentence

File: delivery_dropoff_site/file_exchange.py [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+#  see licence in __openerp__.py or http://www.gnu.org/licenses/agpl-3.0.txt
+#  Copyright (C) 2014 Akretion (http://www.akretion.com).
+#  @author David BEAL <david.beal@akretion.com>
+#
+##############################################################################
+
+from openerp.osv import orm
+import base64
+from cStringIO import StringIO
+import unicodecsv
```
Comment (gurneyalex) [inline delivery_dropoff_site/file_exchange.py:17]: missing external dependency declaration?

File: delivery_dropoff_site/i18n/fr.po [hunk 4]
```diff
@@ -0,0 +1,206 @@
...
+
+#. module: delivery_dropoff_site
+#: help:res.partner,dropoff_site_id:0
+msgid ""
+"... are specific areas where carriers ship merchandises.\n"
+"Recipients comes pick up their parcels in these sites"
+msgstr ""
+"... sont des sites spécifiques où les transporteurs livrent leur "
+"marchandises.\n"
+"Les destinatires viennent récupérer leurs colis dans ces sites."
```
Comment (gurneyalex) [inline delivery_dropoff_site/i18n/fr.po]: s/destinatires/destinataires/ 

did not proof read the whole .po file, this one just caught my eye.

**Comment by gurneyalex** [comment]
missing .pot file

**Comment by bealdav** [comment]
module is updated according to alexandre remarks

**Comment by bealdav** [comment]
This module have a dependency on a non OCA module. Not accepted by OCA

**Pull request closed without merging**